serverless/serverless
# Issue #5619 - Update Cloudflare Templates

## What did you implement:

Closes #5619 



## How did you implement it:



I did the following:
* Bumped the version of the serverless-cloudflare-workers plugin specified in the package.json files of the Cloudflare templates from 0.1.1 to 1.0.5
* Corrected function `name` properties in serverless.yml files of the Cloudflare templates so that the value of the `name` property matches the declared name of the function (serverless-cloudflare-workers throws an exception on `serverless deploy` if these names do not match).
* Updated the docs to reflect this.

## How can we verify it:



To reproduce the issue, you can simply `serverless create --template cloudflare-workers`, update the generated serverless.yml file with your Cloudflare account id, zone id, and zone url (in the `events.http.url` property), then attempt to `serverless deploy`. This will fail with the message "Cannot Convert undefined or null to object".

To verify this fix, I tested locally by creating a new project from this fork, e.g.
```
C:\dev\serverless [master ≡]> node .\bin\serverless create --template cloudflare-workers --path "..\cf-worker-template-test"
```

After adding your account id, zone id, and zone url to the project generated from this fork, `serverless deploy` should successfully deploy your "hello world" worker.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
